docks/base/Dockerfile

```diff
@@ -0,0 +1,27 @@
+## goclang/base
+## an ubuntu-based docker container with go.
+from ubuntu:trusty
+maintainer go-clang-devs@github.com/go-clang
+
+env CODENAME trusty
+env GO_VERSION 1.4.3
+
+run apt-get update -y && \
```

Comment:
Should we include `apt-get upgrade -y` (after update) and `apt-get autoremove -y`(before clean)?